soh/soh/z_message_OTR.cpp

```diff
@@ -19,9 +19,45 @@ MessageTableEntry* OTRMessage_LoadTable(const char* filePath, bool isNES) {
     if (file == nullptr)
         return nullptr;
     
-    MessageTableEntry* table = (MessageTableEntry*)malloc(sizeof(MessageTableEntry) * file->messages.size());
+    MessageTableEntry* table = (MessageTableEntry*)malloc(sizeof(MessageTableEntry) * (file->messages.size() + 1));
 
     for (int i = 0; i < file->messages.size(); i++) {
+        if (file->messages[i].id == 0x2066) {
```

Comment:
Don't we only want to change the options of the owl when `gBetterOwl` is true? In other words, I think there's a missing check here.

Reply:
Additionally, I would add a comment here so people know this has to do with Kaepora.

Reply:
The point with that section of code is to generate new message data based on 0x2066 ahead of time such that it can be swapped to when needed. Unless I'm misunderstanding when OTRMessage_LoadTable is called, if the message data was generated only if gBetterOwl was true, you'd have to relaunch soh every time you toggle it.

I'll push a commit that comments my changes however, seems like this misunderstanding could have been cleared up with them.

Reply:
Ah you're creating a new message. I missed that part. Yeah that shouldn't need a check.